### DISCUSSION TITLE
Please add support for react native windows and react native macos

### DISCUSSION THREAD

**Discussion by hunainhumail-purevpn** [callstack/react-native-builder-bob#449] [Ideas]
I see that this library is supporting android and ios for creating native libraries. But I cant find any method to create native libraries for windows or macos. Please add support for that. Or atleast merge the pull request which is still open in PRs so that we can easily use these features.

**Comment by atlj** [comment]
The problem with React Native Windows and MacOS is the maintenance part. Neither I nor @satya164 are experts in those fields so it would be problematic for us to maintain those platforms at the moment.

**Reply by hunainhumail-purevpn** [reply]
Can you tell me that if I want to do that, then what steps should I follow?
